omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/field/DropDownFragmentController.java

```diff
@@ -19,14 +19,13 @@ public void controller(FragmentConfiguration config) throws Exception {
         if (config.getAttribute("conceptSet") != null) {
             List < Map < String, Object >> options = (List<Map<String, Object>>) config.getAttribute("options");
             if (options == null) {
-            	options = new ArrayList < Map < String, Object >> ();
+            	options = new ArrayList <Map<String, Object>> ();
             }
             String conceptSetUuid = (String) config.getAttribute("conceptSet");
             Concept conceptSet = Context.getConceptService().getConceptByUuid(conceptSetUuid);
 
             for (Concept concept: conceptSet.getSetMembers()) {
-            	Map < String, Object > option = new HashMap < String, Object > ();
-                option = new HashMap < String, Object > ();
+                Map < String, Object > option = new HashMap <String, Object>();
```

Comment:
```suggestion
                Map < String, Object > option = new HashMap < String, Object >();
```
or
```suggestion
                Map <String, Object> option = new HashMap <String, Object>();
```

Try to write consistent code. If you sometimes put spaces and sometimes not, it shows that you don't care about style. Yes, I know, these are trivialities, but it's worth perfecting your art.